test/solvers.jl

```diff
@@ -49,7 +49,7 @@ cbc && push!(lp_solvers, Clp.ClpSolver())
 glp && push!(lp_solvers, GLPKMathProgInterface.GLPKSolverLP())
 ipt && push!(lp_solvers, Ipopt.IpoptSolver(print_level=0))
 eco && push!(lp_solvers, ECOS.ECOSSolver(verbose=false))
-osl && push!(lp_solvers, CoinOptServices.OsilSolver())
+#osl && push!(lp_solvers, CoinOptServices.OsilSolver())
```

Comment:
this probably shouldn't have been buried in a 5k line otherwise-unrelated commit